lib/amazonka-core/src/Amazonka/Data/Log.hs

```diff
@@ -34,10 +35,14 @@ instance ToLog ByteStringBuilder where
   build = id
 
 instance ToLog ByteStringLazy where
-  build = Build.lazyByteString
+  build lbs = case LText.decodeUtf8' lbs of
+    Right t | all Char.isPrint (LText.unpack t) -> Build.lazyByteString lbs
```

Comment:
`text` defines its own versions of `all`, so you don't need to unpack.

Reply:
replaced. thanks!